PlanPluginBridge/src/main/java/com/djrapitops/pluginbridge/plan/luckperms/LuckPermsData.java

```diff
@@ -45,57 +47,73 @@ public InspectContainer getPlayerData(UUID uuid, InspectContainer inspectContain
-                getWithIcon("Group", Icon.called("user-friends").of(Family.SOLID))
+        inspectContainer.addValue(
+                getWithIcon("Groups", Icon.called("user-friends").of(Family.SOLID)),
+                new TextStringBuilder().appendWithSeparators(groups, ", ").build()
         );
-        for (Track track : api.getTracks()) {
-            // reduce is used to get the last element
-            String currentGroup = api.getGroups().stream().map(this::getGroupDisplayName).filter(groups::contains).reduce((first, second) -> second).orElse("None");
-            trackTable.addRow(track.getName(), currentGroup);
+
+        Set<Track> tracks = api.getTracks();
+        if (tracks.size() > 0) {
+            TableContainer trackTable = new TableContainer(
+                    getWithIcon("Track", Icon.called("ellipsis-h").of(Family.SOLID)),
+                    getWithIcon("Group", Icon.called("user-friends").of(Family.SOLID))
+            );
+            for (Track track : tracks) {
+                // reduce is used to get the last element
+                String currentGroup = api.getGroups().stream()
+                                         .map(this::getGroupDisplayName).filter(groups::contains)
+                                         .reduce((first, second) -> second).orElse("None");
+                trackTable.addRow(track.getName(), currentGroup);
+            }
+            inspectContainer.addTable("Tracks", trackTable);
         }
-        inspectContainer.addTable("Tracks", trackTable);
 
         return inspectContainer;
     }
 
     @Override
     public AnalysisContainer getServerData(Collection<UUID> uuids, AnalysisContainer analysisContainer) {
+        // There will *always* be atleast 1 group
         TableContainer groupTable = new TableContainer(
                 getWithIcon("Group", Icon.called("user-friends").of(Family.SOLID)),
                 getWithIcon("Weight", Icon.called("weight-hanging").of(Family.SOLID)),
                 getWithIcon("Permissions", Icon.called("list").of(Family.SOLID))
         );
-        for (Group group : api.getGroups()) {
+
+        for (Group group : api.getGroups().stream().sorted(Comparator.comparing(Group::getName)).collect(Collectors.toList())) {
             OptionalInt weight = group.getWeight();
 
             groupTable.addRow(getGroupDisplayName(group), weight.isPresent() ? weight.getAsInt() : "None", group.getPermissions().size());
         }
         analysisContainer.addTable("Groups", groupTable);
 
-        TableContainer trackTable = new TableContainer(
-                getWithIcon("Track", Icon.called("ellipsis-h").of(Family.SOLID)),
-                getWithIcon("Size", Icon.called("list").of(Family.SOLID))
-        );
-        api.getTracks().forEach(track -> trackTable.addRow(track.getName(), track.getSize()));
+        Set<Track> tracks = api.getTracks();
+        if (tracks.size() > 0) {
```

Comment:
`Collection#isEmpty` would be more efficient than `size > 0`, since `isEmpty` has to only check a single entry.